internal/test_unit/config/TCPairsConfig_ALAL2010

```diff
@@ -128,6 +128,11 @@ watch_warn = {
    time_offset = -14400;
 }
 
```

Comment:
So the STORM_ID is defined as "BBCCYYYY" where BB is the 2-letter basin, CC is the 2-digit storm, and YYYY is the 4 digit year. This [comment](https://github.com/dtcenter/MET/blob/82e5833f443e7e8c1cffe9071887b7a8cb5e3c4c/src/libcode/vx_tc_util/track_info.cc#L1075) followed by this [line of code](https://github.com/dtcenter/MET/blob/82e5833f443e7e8c1cffe9071887b7a8cb5e3c4c/src/libcode/vx_tc_util/track_info.cc#L1097) explains the behavior, but I don't see this detail in the User's Guide.

I don't know how we ended up with that convention. Not sure if we inherited from the original NHC vx code or if I just made it up along the way? We could ask @KathrynNewman, but I'd recommend addressing it outside of this pull request.

Reply:
I think this way of representing all storms (rather than a single storm) was legacy from the NHC verification system, but I'm not 100% - I'll see if I can dig this out and follow-up (outside this PR).